docs/hub/README.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+
+In the Collie Hub you will find all community-curated open-source kit modules that you can use for building your cloud foundation
```

Comment:
s: we need to decide if we call it Collie or collie consistently. Docker does Docker and docker cli IIRC

Reply:
I summon @meshchristina to help us out here with style decision 🔬